src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -0,0 +1,100 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
```

Comment:
Im seeing a right bleed in the figma design. Does the component need to extend into the right column of the outer grid?

Reply:
<img width="510" alt="Screen Shot 2020-11-15 at 3 44 15 PM" src="https://user-images.githubusercontent.com/30575095/99200369-c1af5d00-2759-11eb-83df-9bf43a02a367.png">
<img width="1400" alt="Screen Shot 2020-11-15 at 3 42 34 PM" src="https://user-images.githubusercontent.com/30575095/99200372-c247f380-2759-11eb-8c02-821b589891e2.png">